Cargo.toml

```diff
@@ -19,3 +22,5 @@ rand = "0.5"
 
 [features]
 unsafe_internals = []
+reverse_bits = []
```

Comment:
Are these two lines supposed to be deleted?

Reply:
hmm somehow I missed them

Reply:
I thought you might be testing me :p